src/psp-migration.md

```diff
@@ -0,0 +1,375 @@
+# How to migrate from Kubernetes PSPs to Kubewarden policies?
+Kubewarden can be used to replace all deprecated [PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/). Different from the PSPs, Kubewarden has separated
```

Comment:
I was not trying to tell that Kubewarden is better or worst. It's just how it works. I believe this is a result of the gradual process of creating policies to cover the PSPs configuration. And, I think individual policies allow more granular control of what the operator wanna to validate. But I'm not sure if we should add this discussion in this how-to guide. What do you think @divya-mohan0209 ? Should I explain why we use different policies?

Reply:
I understand your point. What I wanted to understand was if there are any explicit benefits of not clustering policies together in a PSP because we have mentioned that as a statement in this section. If there are none, that is fine. But if we can supplement with a benefit, that'd be good is all.

Reply:
I've pushed one more sentence trying to give some benefits of this approach. Please, give me your feedback.